Source/AtomicJS/Javascript/JSVM.cpp

```diff
@@ -247,6 +247,42 @@ void JSVM::SendJSErrorEvent(const String& filename)
 
 }
 
+int JSVM::GetRealLineNumber(const String& fileName, const int lineNumber) {
+    int realLineNumber = -1;
+    String map;
+    String path = fileName;
+    if (!path.EndsWith(".js.map"))
+        path += ".js.map";
+    SharedPtr<File> mapFile(GetSubsystem<ResourceCache>()->GetFile(path));
```

Comment:
https://github.com/rsredsq/AtomicGameEngine/commit/906d84ad1c56991c7b18a3d7795eef6f12045c13 should fix that